lte/gateway/python/magma/pipelined/ebpf/ebpf_manager.py

```diff
@@ -83,10 +96,11 @@ def get_ebpf_manager(config):
 
 
 class ebpf_manager:
-    def __init__(self, sgi_if_name: str, s1_if_name: str, gw_ip: IPAddress, bpf_ul_file: str = BPF_UL_FILE, bpf_dl_file: str = BPF_DL_FILE):
+    def __init__(self, sgi_if_name: str, s1_if_name: str, gw_ip: IPAddress, bpf_ul_file: str = BPF_UL_FILE, bpf_dl_file: str = BPF_DL_FILE, bpf_path: str = DEFAULT_BPF_PATH):
         self.enabled = True
```

Comment:
Can you pass 'orc8r/gateway/c/common/ebpf/EbpfMap.h'  from the unit tests to avoid file consistency issue in unit test when header file is modified.

Reply:
The issue is when passing the header file from a different dir `(cflags=["-include", DEFAULT_BPF_PATH +BPF_HEADER_FILE])) ` its having issues.